src/linear_algebra/finsupp_vector_space.lean

```diff
@@ -177,23 +170,26 @@ open vector_space
 
 variables {K V : Type u} [field K] [add_comm_group V] [vector_space K V]
 
-lemma cardinal_mk_eq_cardinal_mk_field_pow_dim (h : dim K V < cardinal.omega) :
+lemma cardinal_mk_eq_cardinal_mk_field_pow_dim [finite_dimensional K V] :
```

Comment:
We should make `K` and `V` explicit parameters here, since `rw` is struggling with them in `cardinal_lt_omega_of_finite_dimensional`.

Reply:
I hope this is what you intended.

Reply:
It is exactly what I intended :)